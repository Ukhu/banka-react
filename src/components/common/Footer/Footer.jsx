import React from 'react';

const Footer = () => (
  <footer id="hp-footer">
    <p>Banka, Copyright &copy; 2019</p>
  </footer>
);

export default Footer;
