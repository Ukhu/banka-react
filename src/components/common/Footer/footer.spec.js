import React from 'react';
import { configure, mount } from 'enzyme';

import Adapter from 'enzyme-adapter-react-16';
import Footer from './Footer';

configure({ adapter: new Adapter() });

describe('Footer component <Footer />', () => {
  it('renders correctly', () => {
    const wrapper = mount(
      <Footer />,
    );

    expect(wrapper).toMatchSnapshot();
  });

  it('has the footer tag', () => {
    const wrapper = mount(
      <Footer />,
    );

    expect(wrapper.find('footer')).toHaveLength(1);
  });

  it('has a paragraph with correct text', () => {
    const wrapper = mount(
      <Footer />,
    );

    const footerParagraph = 'Banka, Copyright Â© 2019';

    expect(wrapper.find('p')).toHaveLength(1);
    expect(wrapper.find('p').get(0).props.children).toEqual(footerParagraph);
  });
});
