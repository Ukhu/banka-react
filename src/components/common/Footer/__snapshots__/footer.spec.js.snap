// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component <Footer /> renders correctly 1`] = `ReactWrapper {}`;
